tests/storage/test_extraction.py

```diff
@@ -201,7 +261,104 @@ def test_parameters(client):
         df = pd.read_csv(stream, index_col=0, float_precision="round_trip")
 
         # ERT produces a low-quality version
-        assert_almost_equal(parameters_df[col].values, df.values.flatten(), decimal=4)
+        assert_almost_equal(parameters_df[col].values, df.values.flatten(), decimal=5)
+
+
+def test_observations(client):
```

Comment:
I would also rename this test function to give it more semantics, eg. `test_post_observations`?